### ISSUE TITLE
Allow to set which Azure image are used

### ISSUE THREAD

**Issue by pombredanne** [altendky/romp#55]
The Azure images are fixed and further more set to the latest https://github.com/altendky/romp/blob/8ed5aa48af3c26d7d8456c08d04a9fa039f9be93/src/romp/_matrix.py#L12
For a scenario of wheel building, this is not a great setting as I typically want instead the oldest image possible in order to build more compatible wheels.
To deal with this it should be possible to set which images to use for a given OS.

**Comment by pombredanne** [comment]
This becomes pressing because of https://github.com/actions/virtual-environments/issues/1816

**Comment by altendky** [comment]
It might involve a combination, but for Linux you should build in some manylinux if you are compiling.  If not compiling, then it shouldn't matter?  Anyways, along with this probably use cibuildwheel.

https://github.com/joerick/cibuildwheel

**Comment by pombredanne** [comment]
@altendky exactly. Being able to set both the Azure VM image and a Docker image that's used on Linux (e.g. one of the manylinux from cibuildwheel would make sense

**Comment by pombredanne** [comment]
in fact ideally I would want to run @joerick cibuildwheel as a cli under romp, since my use case is to build some arbitrary wheels on many Python/OS combos (and NOT to build a certain repo as wheels on many Python/OS combos for which cibuildwheels would be all I need)
